src/pages/mine.tsx

```diff
@@ -32,11 +32,11 @@ const Mine: FC = () => {
   const { isLoggedIn, isLoading: isAuthenticating } = useAuthenticated();
 
   useEffect(() => {
-    if (isLoggedIn && !Object.keys(response).length) {
+    if (isLoggedIn) {
```

Comment:
since you removed length from the if condition, how are you planning to handle users with 0 tasks?

Reply:
Also please update the tests in case they were supposed to

Reply:
@RitikJaiswal75 , @kshitij430 has added description addressing it

Reply:
Test cases would have helped here

Reply:
Can you point us to a resource that can help in writing test for this scenario?